programs/tests/mt-c-tests/uart/uart.c

```diff
@@ -0,0 +1,236 @@
+/**
+ * @author Magnus Mæhlum (magnmaeh@stud.ntnu.no)
+ * @brief This test implements a Software-defined UART, which is a more formal
```

Comment:
Very cool. The SD UART should be a library that the user can include. Look at how XMOS has done it: https://github.com/orgs/xmos/repositories?page=2